reflectapi-demo/src/lib.rs

```diff
@@ -91,6 +91,8 @@ mod model {
         #[serde(default, skip_serializing_if = "Option::is_none")]
         #[deprecated(note = "test deprecation")]
         pub age: Option<u8>,
+        #[serde(default = "super::some_datetime")]
```

Comment:
+ skip_serializing_if?

Reply:
If what though? It will always exist on output.